package.xml

```diff
@@ -2,7 +2,7 @@
 <package format="2">
   <name>robot_localization</name>
   <version>2.3.1</version>
-  <description>The robot_localization package provides nonlinear state estimation through sensor fusion of an abritrary number of sensors.</description>
+  <description>Nonlinear state estimation through sensor fusion of an abritrary number of sensors.</description>
```

Comment:
I shorten it, because one of the linter was complaining the description is starting with a boilerplate or something similar.